src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -587,9 +587,27 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
       return false;
     }
     const hearingWindowHMC = this.hearingRequestMainModel.hearingDetails.hearingWindow;
+
+    if (this.hearingRequestMainModel.partyDetails) {
+      if (HearingsUtils.hasPartyUnavailabilityDatesChanged(this.hearingRequestToCompareMainModel.partyDetails, this.serviceHearingValuesModel.parties)){
+        return true;
+      }
+    }
     if ((hearingWindowHMC?.firstDateTimeMustBe) ||
       (hearingWindowHMC?.dateRangeStart || hearingWindowHMC?.dateRangeEnd)) {
-      return HearingsUtils.hasPartyUnavailabilityDatesChanged(this.hearingRequestToCompareMainModel.partyDetails, this.serviceHearingValuesModel.parties);
+      if (HearingsUtils.hasHearingDatesChanged(this.hearingRequestMainModel.hearingDetails.hearingWindow, this.serviceHearingValuesModel.hearingWindow)){
+        return true;
+      }
+    }
+    if (this.hearingRequestMainModel.hearingDetails.duration){
+      if (HearingsUtils.hasHearingDurationChagned(this.hearingRequestMainModel.hearingDetails.duration, this.serviceHearingValuesModel.duration)){
```

Comment:
small spelling mistake here, which normally i wouldnt mention

Reply:
Resolved, thank you